android/src/main/java/com/swmansion/reanimated/layoutReanimation/ReanimatedNativeHierarchyManager.java

```diff
@@ -100,14 +100,32 @@ public void applyLayoutUpdate(View view, int x, int y, int width, int height) {
    */
   public void deleteView(final View view, final LayoutAnimationListener listener) {
     UiThreadUtil.assertOnUiThread();
+    NativeViewHierarchyManager nativeViewHierarchyManager = mWeakNativeViewHierarchyManage.get();
+    ViewManager viewManager = nativeViewHierarchyManager.resolveViewManager(view.getId());
+    // we don't want layout animations in native-stack since it is currently buggy there
+    // so we check if it is a (grand)child of ScreenStack
+    if (viewManager instanceof ViewManager
```

Comment:
I think we can just check if it is not `null` since the method above returns `ViewManager` or `null`. Am I right?

Reply:
I have just checked and `resolveViewManager` never returns null. It throws an exception if don't recognize the tag.
https://github.com/facebook/react-native/blob/v0.67.0-rc.3/ReactAndroid/src/main/java/com/facebook/react/uimanager/NativeViewHierarchyManager.java#L108-L115

Reply:
Should we add `try {} catch`? What do you think?

Reply:
I think we can do something similar to this: https://github.com/facebook/react-native/blob/v0.67.0-rc.3/ReactAndroid/src/main/java/com/facebook/react/uimanager/NativeViewHierarchyManager.java#L137. I can see there is no `try-catch` in `manageChildren` in RN code, so prolly no point in using it there, but it seems reasonable to add it in `deleteView`